modules/datadict/ajax/UpdateDataDict.php

```diff
@@ -37,32 +37,32 @@
 } else {
     // Magic quotes is off, so we can just directly use the description
     // since insert() will use a prepared statement.
-    $description = $_REQUEST['description'];
+    $description = trim($_REQUEST['description']);
```

Comment:
PHP hasn't supported magic quotes for a fairly long time now (it was removed in PHP 5.4, and the minimum requirement for LORIS is PHP 7.1..)

I think this if statement can just be removed.